### PR TITLE
Winlogbeat - Select events by level, event_id, and provider

### PR DESCRIPTION
**Pull request by andrewkroh**
This PR adds the ability to do select specific events from a single log. This is implemented using [an event log query](https://msdn.microsoft.com/en-us/library/windows/desktop/dd996910(v=vs.85).aspx).

Sample Winlogbeat Config:
```
winlogbeat:
  event_logs:
    - name: Security
      include_xml: true
      level: info
      event_id: 4624, 4625, 4648, 4735, 4728, 4732, 4756, 4740
      provider:
        - Microsoft-Windows-Security-Auditing
```

XPath Query Used by Winlogbeat:

```
<QueryList>
  <Query Id="0">
    <Select Path="Security">*[System[(EventID=4624 or EventID=4625 or 
                  EventID=4648 or EventID=4735 or EventID=4728 or EventID=4732 or 
                  EventID=4756 or EventID=4740) and (Level = 0 or Level = 4) and 
                  Provider[@Name='Microsoft-Windows-Security-Auditing']]]
    </Select>
  </Query>
</QueryList> 
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,6 @@ env:
     - TARGETS="-C libbeat testsuite"
     - TARGETS="-C topbeat testsuite"
     - TARGETS="-C filebeat testsuite"
-    - TARGETS="-C winlogbeat testsuite"
```
Comment (ruflin) [inline .travis.yml]: Is there a reason to completely remove it?

**Comment by ruflin** [comment]
Doesn't this require also an updates to `beat.yml` ?

File: .travis.yml (hunk 1 above)
Comment (andrewkroh) [inline .travis.yml]: The system tests are all (now) skipped unless the OS is windows so I want it removed to use less Travis resources.

**Comment by andrewkroh** [comment]
Yeah, I should update `beat.yml` with the new options. :+1:  Will do.

**Comment by andrewkroh** [comment]
I updated the config file to include a mention of all the configuration options and added a link to the documentation.

**Comment by tsg** [comment]
LGTM, merging.

**Pull request merged**